test/Microsoft.AspNetCore.WebSockets.ConformanceTest/Autobahn/AutobahnTester.cs

```diff
@@ -65,11 +67,12 @@ public AutobahnTester(int startPort, ILoggerFactory loggerFactory, AutobahnSpec
             }
         }
 
-        public void Verify(AutobahnResult result)
+        public void Verify(AutobahnResult result, CancellationToken cancellationToken)
         {
             var failures = new StringBuilder();
             foreach (var serverResult in result.Servers)
             {
+                cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Yeah, that's probably unnecessary. Just getting a little trigger-happy with the `ThrowIfCancellationRequested`